### PR TITLE
Promote next event

### PR DESCRIPTION
**Pull request by jburel**
Promote next workshop 
This PR also adds links to the 2 GBI events

### CODE REVIEW

**Comment by jburel** [comment]
See https://snoopycrimecop.github.io/www.openmicroscopy.org/

**Comment by pwalczysko** [comment]
All looks good, but maybe i do not understand - Basel is the next event (still as of now). This will be deployed on Monday or Wednesday ?

**Comment by jburel** [comment]
It will be deployed mid-week post Basel

**Comment by pwalczysko** [comment]
Ready to merge then. Thank you

**Comment by will-moore** [comment]
Yep - looks good.

File: events/index.html [hunk 1]
```diff
@@ -34,6 +36,19 @@ <h2><a href="https://miap.eu/index.php?id=446" target="_blank">OMERO Workshop, B
                 <p class="card-caption">Two full-day workshops for new and advanced users and sysadmins.</p>
             </div>
             <hr class="medium-8">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> July 09-14, 2018</h6>
+                <h2><a href="http://www.e-smi.eu/index.php?id=topim-tech">OMERO Workshop, Chania</a></h2>
+                <p class="card-caption">Topim Tech: A full day workshops about the OMERO ecosystem.</p>
```
Comment (mtbc) [inline events/index.html]: add "of"

Comment (will-moore) [reply events/index.html]: Or "A full-day workshop about..."

**Comment by jburel** [comment]
@will-moore @mtbc updated

**Comment by jburel** [comment]
Merging so we can update front page

**Pull request merged**